myParser.py

```diff
@@ -40,6 +40,12 @@ def parse():
         required=True,
         help="Date of departure flight.")
     
+    parser.add_argument(
```

Comment:
2 things:

1. Can you change "check" to "track" and add a period at the end. I think that makes more sense.
2. Can you edit the Usage section of the README to give more information about this command?

Reply:
Will do